apps/laboratory/tests/email.spec.ts

```diff
@@ -62,39 +63,41 @@ emailTest('it should reject sign', async () => {
   await validator.expectRejectedSign()
 })
 
-emailTest('it should switch network and sign', async () => {
+emailTest('it should switch network and sign', async ({ library }) => {
   let targetChain = 'Polygon'
-  await page.openAccount()
-  await page.openProfileView()
-  await page.openSettings()
+  await page.goToSettings()
   await page.switchNetwork(targetChain)
+  if (library === 'wagmi') {
+    // In wagmi, after switching network, it closes the modal
```

Comment:
We can fix later, let's create a card for it, merge this and fix if there's an issue.
Only happens in tests?

Reply:
This is happening when using Wagmi, but it's not a breaking changes since just affecting the UX. Created the ticket here: https://linear.app/walletconnect/issue/APKT-899/in-wagmi-email-switching-network-closing-the-modal-unlike-the